iaso/models/org_unit.py

```diff
@@ -280,6 +280,9 @@ class OrgUnit(TreeModel):
     updated_at = models.DateTimeField(auto_now=True)
     creator = models.ForeignKey(User, null=True, blank=True, on_delete=models.SET_NULL)
 
+    opening_date = models.DateField(blank=False, null=True)  # Start date of activities of the organisation unit
```

Comment:
It's not a good idea to make this mandatory: this would break all existing mobile applications that are unaware of that field, and also a lot of api integrations that are not providing that information

Reply:
@madewulf solved